cmd/main.go

```diff
@@ -89,6 +90,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	if !utils.IsEnvironmentVariablesExist() {
+		setupLog.Error(err, "failed to get configuration environment variable")
+		os.Exit(1)
+	}
+	if !utils.IsReclaimPolicyValid(os.Getenv("RECLAIM_POLICY")) {
```

Comment:
Maybe put all the checks for environment variables into one function, and return false/true one time with the corresponding error message(like the flow of validating webhook for example)